chapter_convolutional-modern/resnet_vn.md

```diff
@@ -344,8 +345,10 @@ d2l.train_ch6(net, train_iter, test_iter, num_epochs, lr)
 * ResNet had a major influence on the design of subsequent deep neural networks, both for convolutional and sequential nature.
 -->
 
-*dịch đoạn phía trên*
-
+* Các khối thặng dư cho phép tham số hóa tương đối đến hàm đồng nhất $f(\mathbf{x}) = \mathbf{x}$.
```

Comment:
```suggestion
* Các khối thặng dư cho phép tham số hóa đến hàm đồng nhất $f(\mathbf{x}) = \mathbf{x}$.
```
chắc vẫn cần thống nhất thiêm việc dịch "residual" block. 
"relative to" theo mình ở đây có nghĩa là liên quan tới, chứ không phải là "tương đối". Nếu nghĩa là "tương đối cần đặt lên trước từ parametrization".

Reply:
Dạ, nếu ý họ là liên quan tới thì có lẽ họ đã dùng từ `related to` rồi anh nhỉ? Em thấy một vài tính từ hay có cách dùng phổ biến là N + adj  thay vì chỉ là adj + N.

Reply:
Đội viết quyển này là đội TQ, viết bản gốc đọc cũng hơi bị ngang. Vì nếu em dùng từ "tương đối" thì phải dùng relatively và đặt trước mới chuẩn. Và nếu em dùng "tương đối" thì cái gì ở đây là tương đối?

Tham số hóa tương đối là như thế nào?

Reply:
Dạ vâng, em đồng ý ạ. commit đầu e cũng dịch là liên quan nhưng được góp ý lại là "tương đối" sát với từ gốc đó nên em sửa vì thấy hợp lý chứ nghĩa của câu thì k xuôi lắm.